src/wiz.c

```diff
@@ -735,16 +797,56 @@ static void wiz_link_status_thread_entry(void *parameter)
                 LOG_I("%s netdev link status becomes link down", wiz_netdev_name);
             }
         }
-        rt_thread_mdelay(1000);
+        rt_thread_mdelay(4000);
```

Comment:
i think 1000 will be more quick.

Reply:
```c
#ifdef WIZ_USING_DHCP
    /* alloc IP address through DHCP */
    {
        result = wiz_network_dhcp(netdev);
        if (result != RT_EOK)
        {
            b_status = RT_FALSE;
            LOG_E("WIZnet network initialize failed, DHCP timeout.");
        }
        else
        {
            b_status = RT_TRUE;
            LOG_D("WIZnet network initialize success.");
        }
    }
```

I think the log is not suitable, because network haven't been initiated at this time.

Reply:
> i think 1000 will be more quick.

网线状态是低优先级事件，在共用总线时1秒浪费带宽

Reply:
> ```c
> #ifdef WIZ_USING_DHCP
>     /* alloc IP address through DHCP */
>     {
>         result = wiz_network_dhcp(netdev);
>         if (result != RT_EOK)
>         {
>             b_status = RT_FALSE;
>             LOG_E("WIZnet network initialize failed, DHCP timeout.");
>         }
>         else
>         {
>             b_status = RT_TRUE;
>             LOG_D("WIZnet network initialize success.");
>         }
>     }
> ```
> 
> I think the log is not suitable, because network haven't been initiated at this time.

这一处先留着等我下次更新吧